src/deepsparse/yolo/pipelines.py

```diff
@@ -185,6 +185,12 @@ def process_inputs(
 
         image_batch = list(self.executor.map(self._preprocess_image, inputs.images))
 
+        original_image_shapes = None
+        if image_batch and isinstance(image_batch[0], tuple):
```

Comment:
When would `image_batch` be a tuple?

Reply:
Each item in image batch list would now be a tuple since `_preprocess_image` is now also returning the original input shape